tools/sst_dump_tool.cc

```diff
@@ -280,8 +280,7 @@ int SSTDumpTool::Run(int argc, char const* const* argv, Options options) {
       print_help(/*to_stderr*/ false);
       return 0;
     } else if (strcmp(argv[i], "--version") == 0) {
-      printf("sst_dump from RocksDB %d.%d.%d\n", ROCKSDB_MAJOR, ROCKSDB_MINOR,
-             ROCKSDB_PATCH);
+      printf("sst_dump:\n%s\n", GetRocksBuildPropertiesAsString().c_str());
```

Comment:
This should mention RocksDB, so that people can figure out what project / purpose sst_dump serves just by running --version or --help.

    $ sst_dump --version
    sst_dump from RocksDB 6.15.1
    $ cat --version
    cat (GNU coreutils) 8.30
    $ zcat --version
    zcat (gzip) 1.9

I think the parentheses are a good format to use.

Reply:
$ ./sst_dump --version
sst_dump (RocksDB) 6.16.0

Reply:
Done